.github/workflows/ci_checks.yml

```diff
@@ -125,6 +125,14 @@ jobs:
       - name: Verify existing Room schemas are not modified
         run: current/.github/scripts/compare_directory_contents.sh master/app/schemas/org.simple.clinic.AppDatabase current/app/schemas/org.simple.clinic.AppDatabase
 
+  verify_pr_description:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
We won't need to checkout the code since this step only needs access to the PR meta-data.

Reply:
I was using that to use the shell script in the scripts folder. But since this is just a simple regex check I can move the scrip to yaml file itself. wdyt?

Reply:
Ah, I see. Moving the regex to the YAML seems fine.